File: src/main/java/com/back/nbe9112team06/Nbe9112Team06Application.java
package com.back.nbe9112team06;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;

@SpringBootApplication
@EnableJpaAuditing
public class Nbe9112Team06Application {

    public static void main(String[] args) {
        SpringApplication.run(Nbe9112Team06Application.class, args);
    }

}

File: src/main/java/com/back/nbe9112team06/Nbe9112Team06Application.kt
package com.back.nbe9112team06

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.data.jpa.repository.config.EnableJpaAuditing

@SpringBootApplication
@EnableJpaAuditing
class Nbe9112Team06Application
    fun main(args: Array<String>) {
        runApplication<Nbe9112Team06Application>(*args)
    }

